content/roadmaps/114-defi/content/110-borrowing/106-aave.md

```diff
@@ -1 +1,8 @@
 # AAVE
+
+Aave is a decentralized non-custodial liquidity protocol where users can participate as depositors or borrowers. Depositors provide liquidity to the market to earn a passive income, while borrowers are able to borrow in an overcollateralized (perpetually) or undercollateralized (one-block liquidity) fashion.
+Aave suports Ethereum, Ploygon, Optimism, Arbitrum, Fantom, Harmony and Avalanche chains.
+
+<ResourceGroupTitle>Free Content</ResourceGroupTitle>
+<BadgeLink badgeText='Read' colorScheme='yellow' href='https://docs.aave.com/hub/'>Aave Document Hub</BadgeLink>
+<BadgeLink badgeText='Read' colorScheme='yellow' href='https://decrypt.co/resources/what-is-aave-inside-the-defi-lending-protocol'>What Is Aave? Inside the DeFi Lending Protocol</BadgeLink>
```

Comment:
Please review a bit resources from Murathan available on: https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&p=props&layers=1&nav=1&title=defi_developer_roadmap%40latest_xml.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1rLrEkwj9BLhopoHOOg4BGI_kYi63aSud%26export%3Ddownload

Reply:
Keep in mind that last thing which we want to link to is official project documentation. We probably would like to focus on some technical summaries for protocols